CoreDataAccessView/src/au/gov/asd/tac/constellation/views/dataaccess/panes/DataAccessViewCategoryPanelController.java

```diff
@@ -73,6 +77,7 @@ public void applyChanges() {
                 final Preferences prefs = NbPreferences.forModule(DataAccessViewPreferenceKeys.class);
                 final DataAccessViewCategoryPanel panel = getPanel();
                 prefs.put(DataAccessViewPreferenceKeys.HIDDEN_DA_VIEW, panel.getHiddenCategory().toString().replace("[,", "["));
+                prefs.put(DataAccessViewPreferenceKeys.VISIBLE_DA_VIEW, panel.getVisibleCategory().toString().replace("[,", "["));
                 panelRefreshed = true;
```

Comment:
After this line is where you should be setting the ordering variables back to false:
```
                orderChanged = false;
                reorderButtonPressed = false;
```